examples/examples_nodejs_test.go

```diff
@@ -21,16 +20,9 @@ func TestAccService(t *testing.T) {
 }
 
 func TestAccPgService(t *testing.T) {
-	t.Skip() // due to https://github.com/pulumi/pulumi-aiven/issues/235
 	test := getJSBaseOptions(t).
 		With(integration.ProgramTestOptions{
 			Dir: path.Join(getCwd(t), "pg-service"),
-			EditDirs: []integration.EditDir{
```

Comment:
Was this EditDirs check causing trouble that we intentionally remove it?

Reply:
The removed test relied on a removed `Service` Resource. That was removed in our v5.